app/views/PenalisedEstateAgentsActView.scala.html

```diff
@@ -34,10 +34,20 @@
 
         @components.error_summary(form.errors)
 
+        @components.page_heading("eab.heading")
+
+        @components.heading("penalisedEstateAgentsAct.heading")
+
+        <p>@Messages("penalisedEstateAgentsAct.subtitle")</p>
+        <ul class="list list-bullet">
+            <li>@Messages("penalisedEstateAgentsAct.warning")</li>
+            <li>@Messages("penalisedEstateAgentsAct.banned")</li>
+        </ul>
+
         @components.input_yes_no(
             field = form("value"),
             legend = messages("penalisedEstateAgentsAct.heading"),
-            headingIsLegend = true
+            legendClass = Some("visually-hidden")
```

Comment:
I think that headingIsLegend = true will be necessary here, we can discuss tomorrow morning.

Reply:
Done ^ also good job adding supportingContent. Fixes issues with tests and content being displayed below header :+1: